action/stake_create.go

```diff
@@ -30,28 +29,23 @@ const (
 type CreateStake struct {
 	AbstractAction
 
-	candAddress address.Address
-	amount      *big.Int
-	duration    uint32
-	autoStake   bool
-	payload     []byte
+	candName  string
```

Comment:
i recall we discussed using CandName here?

then we can use it to directly access the delegate map